app/models/api-comments.model.js

```diff
@@ -40,3 +40,23 @@ RETURNING *;`, [username, body, article_id])
 })
 }
 
+exports.removeComment = (comment_id) => {
+    return db.query(`SELECT * FROM comments
+    WHERE comment_id = $1`, [comment_id])
+    .then((response) => {
+        if (response.rows.length === 0){
+            return Promise.reject({status: 404, message: 'No comment found'})
+        } else {
+            return db.query(`DELETE FROM comments
+            WHERE comment_id = $1
+            RETURNING *`, [comment_id])
+        }
+    })
+    // .then(() => {
+    //     return db.query(`SELECT * FROM comments
+    //     WHERE comment_id = $1`, [comment_id])
+    // })
+    // .then((response) => {
+    //     console.log(response.rows)
+    // })
+    }
```

Comment:
If you commented out code, this should be removed when comitting and pushing. Creating a PR for others to review means "I have finished this piece of work", and there's no good justification for this still being here